ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts

```diff
@@ -255,8 +256,8 @@ export class RemoteEnvironmentService extends EnvironmentService {
             path.join(this.experimentRootDir, "environment-temp")
         await executor.createFolder(environment.runnerWorkingFolder);
         await execMkdir(environmentLocalTempFolder);
-        await fs.promises.writeFile(path.join(environmentLocalTempFolder, executor.getScriptName("run")),
-        environment.command, { encoding: 'utf8' });
+        await createScriptFile(path.join(environmentLocalTempFolder, executor.getScriptName("run")),
+                environment.command);
```

Comment:
Same here.

Reply:
The old indent was wrong.
8 spaces indent and parentheses indent are both acceptable. Since I'm the only one maintaining this code, I guess I can choose my preferred style.